pkg/ddevapp/app_compose_template.yaml

```diff
@@ -45,6 +45,7 @@ services:
     environment:
       - COLUMNS
       - DDEV_DATABASE
+      - DDEV_DATABASE_FAMILY
```

Comment:
Also add it bellow to the web container (~line 172)

Reply:
Fixed in #0203dcc6bd8d1382d7507d47707e4a88aa91aab2